src/safe.css

```diff
@@ -11,6 +11,6 @@ body {
     width: 100%;
 }
 
-iframe {
+body > iframe {
     display: none;
```

Comment:
Can possibly even removed, as the new Hub and Network clients should add this on the iframe tags themselves. But doesn't hurt to leave it here.